docs/jade-getting-started.md

```diff
@@ -262,6 +265,21 @@ create extension pgcrypto;
   * Google Project Name: `broad-jade-ZZ` (replacing `ZZ` with your initials)
   * Google Organization: broadinstitute.org
 
+6. `ctrl-d` out of the psql shell and run the following to connect to your personal instance database:
+
+```
+cd jade-data-repo/ops
+DB=datarepo-ZZ SUFFIX=ZZ ENVIRONMENT=dev ./db-connect.sh
```

Comment:
This is a bit awkward as you've already run `cd` above. If the two commands in these steps were changed to be run from the root directory it might be less awkward.
```suggestion
DB=datarepo-ZZ SUFFIX=ZZ ENVIRONMENT=dev ops/db-connect.sh
```
(the `./` prefix is only needed if the command is in the current directory)